s3/pkg/datastore/yig/config/config.go

```diff
@@ -135,21 +135,32 @@ func (cc *CacheConfig) Parse(vals map[string]interface{}) error {
 }
 
 type DatabaseConfig struct {
-	DbType     int
-	DbUrl      string
-	DbPassword string
+	DbType       string
+	DbUrl        string
+	DbPassword   string
+	MaxIdleConns int
+	MaxOpenConns int
 }
 
 func (dc *DatabaseConfig) Parse(vals map[string]interface{}) error {
+	dc.MaxIdleConns = 0
```

Comment:
why is this 0 ? how will connection pooling work with no idle connections ?

Reply:
@rhsakarpos thanks for your nicely review, and i have changed the value to a macro which by default is 1024. both max idle conns and max open conns are 1024, and both values are defined in separate macro.